pulp_ansible/app/tasks/collections.py

```diff
@@ -112,55 +114,30 @@ def import_collection(
             the Collection's metadata. If it does not match a ValidationError is raised.
 
     Raises:
-        ValidationError: If the `expected_namespace`, `expected_name`, or `expected_version` do not
+        ImporterError: If the `expected_namespace`, `expected_name`, or `expected_version` do not
```

Comment:
It's also above in the docstring in more places (per attribute). Removing it from above would also be ok w/ me.

Reply:
Done